cpp/build/support.mk

```diff
@@ -96,9 +107,15 @@ else
 pkgconfigdir ?= $(shell pkg-config --variable pc_path pkg-config | awk '{split($$0,a,":"); print a[1]}')
 endif
 
+ifeq ($(BITBAKE_ENV),1)
+sysconfdir := $(PREFIX)/etc/openzwave/
+includedir := $(PREFIX)/include/openzwave/
+docdir := $(PREFIX)/share/doc/openzwave-$(VERSION).$(VERSION_REV)
+else
 sysconfdir ?= $(PREFIX)/etc/openzwave/
```

Comment:
In the yocto bitbake env, the support.mk being involved multiple times.
When the first time being involved, the "PREFIX" is empty and
"

sysconfdir ?= $(PREFIX)/etc/openzwave

"
will keep the value //etc/openzwave.

With the ":=", the build will be able to pick up the up to date PREFIX 
setting as the build infrastructure specified, and doing right install.

Regards
Bin


On 16-04-25 01:10 AM, Justin Hammond wrote:
>
> In cpp/build/support.mk 
> <https://github.com/OpenZWave/open-zwave/pull/861#discussion_r60863169>:
>
> > @@ -96,9 +107,15 @@ else
> >  pkgconfigdir ?= $(shell pkg-config --variable pc_path pkg-config | awk '{split($$0,a,":"); print a[1]}')
> >  endif
> >
> > +ifeq ($(BITBAKE_ENV),1)
> > +sysconfdir := $(PREFIX)/etc/openzwave/
> > +includedir := $(PREFIX)/include/openzwave/
> > +docdir := $(PREFIX)/share/doc/openzwave-$(VERSION).$(VERSION_REV)
> > +else
> >  sysconfdir ?= $(PREFIX)/etc/openzwave/
>
> why this? they are the same as the defaults....
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub 
> <https://github.com/OpenZWave/open-zwave/pull/861/files/8c26a41e9f9f3a4a9484c53194c0b3ca7721db4b#r60863169>
>